lib/iris/tests/results/netcdf/int64_data_netcdf3.cml

```diff
@@ -6,6 +6,6 @@
     </attributes>
     <coords/>
     <cellMethods/>
-    <data checksum="0xecbb4b55" dtype="int32" shape="(2, 2)"/>
+    <data checksum="0xecbb4b55" dtype="int32" mask_checksum="no-masked-elements" shape="(2, 2)"/>
```

Comment:
I guess that this change is needed due to https://github.com/Unidata/netcdf4-python/pull/787